p2p/src/main/java/bisq/network/p2p/peers/getdata/RequestDataHandler.java

```diff
@@ -256,13 +250,21 @@ private void logContents(GetDataResponse getDataResponse) {
         numPayloadsByClassName.forEach((key, value) -> sb.append(key)
                 .append(": ")
                 .append(value.first.get())
-                .append(" / ≈")
-                .append(Utilities.readableFileSize(value.second * value.first.get()))
+                .append(" / ")
+                .append(Utilities.readableFileSize(value.second.get()))
                 .append(sep));
         sb.append("#################################################################");
         log.info(sb.toString());
     }
 
+    private void addDetails(Map<String, Tuple2<AtomicInteger, AtomicInteger>> numPayloadsByClassName,
+                            NetworkPayload networkPayload, String className) {
+        numPayloadsByClassName.putIfAbsent(className, new Tuple2<>(new AtomicInteger(0),
+                new AtomicInteger(0)));
+        numPayloadsByClassName.get(className).first.getAndIncrement();
+        numPayloadsByClassName.get(className).second.getAndAdd(networkPayload.toProtoMessage().getSerializedSize());
```

Comment:
Might be worth adding a comment that this log info comes at a cost for in case it turns out to be a problem in the future

Reply:
Yes can add a comment